### ISSUE TITLE
[[ " $var "  ]] breaks syntax highlighting

### ISSUE THREAD

**Issue by ItamarShDev** [jeff-hykin/better-shell-syntax#18]
![image](https://user-images.githubusercontent.com/953371/139854880-34143cda-a8b3-460d-b9ad-d7112caa0606.png)

**Comment by varenc** [comment]
I'm getting this too! Any space in any side of the `=~'  condition breaks the syntax checking for the code below it.  Here's a simpler example that breaks it more catastrophically:

```sh
[[ $T =~ "a string" ]] && echo "yes" || echo "no""
function testo() {  ### this loses highlighting due to the space in "a string" above
    ls -la
    pwd
}
```

<img width="1015" alt="image" src="https://user-images.githubusercontent.com/167585/197086923-d6606803-c644-4070-b3c2-7158538959f2.png">


removing the space in `a string` makes it work correctly.

**Comment by jeff-hykin** [comment]
should now be fixed! `v1.2.5`

**Comment by varenc** [comment]
thank you!! though another issue: the latest version, v1.2.9, now seems to cause syntax highlighting breakage on this code:

    1="`mpp`"

All the function after that don't get blue names and other highlighting breaks.

Of course re-assigning the `$1` var is bad practice so I don't mind too much this reminder to fix that.  Was confusing that this new issue popped up but this note made me realize I'd been auto-updated.  Downgrading to v1.1.0 fixed it.
